tests/library/Garp/Content/CDN/AssetListTest.php

```diff
@@ -44,14 +44,17 @@ public function test_One_Garp_Asset_Should_Be_Selected_If_Specific_Match() {
         $this->assertSame(count($assetList), 1);
     }
 
-    public function test_One_App_Asset_Should_Be_Selected_If_Specific_Match() {
+    //public function test_One_App_Asset_Should_Be_Selected_If_Specific_Match() {
```

Comment:
Shouldn't we just remove this test? Don't see much added value as it is.

Reply:
Absolutely. Done!